src/Windowing/Silk.NET.Windowing.Desktop/GlfwWindow.cs

```diff
@@ -152,6 +153,31 @@ public bool IsVisible
         /// </remarks>
         public bool ShouldSwapAutomatically { get; }
 
+        /// <inheritdoc />
+        public unsafe VideoMode VideoMode
+        {
+            get
+            {
+                var monitor = _glfw.GetWindowMonitor(_windowPtr);

```

Comment:
Will this throw an error if `_windowPtr` is invalid or a nullptr?

Reply:
As I changed the whole property implementation this is no longer an issue.